robottelo/config/base.py

```diff
@@ -552,14 +553,15 @@ def read(self, reader):
         self.hostname_ipa = reader.get('ipa', 'hostname_ipa')
         self.password_ipa = reader.get('ipa', 'password_ipa')
         self.username_ipa = reader.get('ipa', 'username_ipa')
+        self.user_ipa = reader.get('ipa', 'user_ipa')
```

Comment:
Alert! You have not added this property/variable in robottelo.properties.sample

Reply:
Added!